lib/validation/validators/negativeInteger.ts

```diff
@@ -1,3 +1,9 @@
-export function negativeInteger(value: number): boolean {
+import { isNumber } from '../../support/Utils'
+
+export function negativeInteger(value: unknown): boolean {
+  if (!isNumber(value)) {
+    return false
+  }
+
   return Number.isInteger(value) && value < 0
 }
```

Comment:
`isNumber` guard is needed because `Number.isInteger()` static method just returns boolean without type predicate (i.e. `value is number`).